### PR TITLE
Update see also code comment to @see

### PR DESCRIPTION
**Pull request by tungbq**
Closes: #292

### CODE REVIEW

**Comment by bitspittle** [comment]
Wow, thanks for the quick help on this!

I did a quick browse and noticed that at least in one place you replaced some inline "see also" comments which we probably don't want to do. Would you like to review your own code, or would you like me to go through and leave comments for the places we should probably revert?

**Comment by bitspittle** [comment]
Also, side note that we don't actually work out of the main branch (see also: https://github.com/varabyte/kobweb/blob/main/CONTRIBUTING.md#latest-branch) but rather the latest version branch (which in this case would be 0.13.11).

That said, I appreciate your help, so I can handle any rebase conflicts that might happen if you'd prefer.

**Comment by tungbq** [comment]
Thanks for your promptly response @bitspittle and also the side note you provided.
I will update the development branch as per suggested. And also review my own code first, then will need your review at the second gate. Thanks!

**Comment by tungbq** [comment]
Hi @bitspittle, I've self-reviewed and just updated the PR to keep the inline 'see also' as is. Could you please have a look? Thanks

**Review by bitspittle — COMMENTED** [review]
(Whoops I forget to send out the comments, sorry about that!)

**Comment by bitspittle** [comment]
BTW @tungbq just a heads up that we're going to release 0.13.11 today (unless something crazy happens in the next few hours). 

This shouldn't affect you too much except we'll probably have to rebase your commits against a new branch (which doesn't exist yet). I'll update this comment when this happens.

**Comment by bitspittle** [comment]
OK, just wanted to update that 0.13.11 is out, so dev has moved to 0.13.12. You'll probably need to rebase on top of it, but hopefully it's trivial. Apologies for this happening in the middle of your review!

**Comment by tungbq** [comment]
> OK, just wanted to update that 0.13.11 is out, so dev has moved to 0.13.12. You'll probably need to rebase on top of it, but hopefully it's trivial. Apologies for this happening in the middle of your review!

Hey @bitspittle you are very nice. Thanks for all your comments, it's very informative.
I've just logged in on my time zone. I will check all your comments then. Thanks!

File: backend/kobweb-api/src/main/kotlin/com/varabyte/kobweb/api/http/Request.kt [hunk 1]
```diff
@@ -20,7 +20,7 @@ package com.varabyte.kobweb.api.http
  * }
  * ```
  *
- * See also: [Response]
+ * @see: [Response]
```
[the remark this answers is not in the document]
Comment (tungbq) [reply backend/kobweb-api/src/main/kotlin/com/varabyte/kobweb/api/http/Request.kt]: Updated

File: backend/kobweb-api/src/main/kotlin/com/varabyte/kobweb/api/http/Response.kt [hunk 2]
```diff
@@ -16,13 +16,13 @@ val EMPTY_BODY = ByteArray(0)
...
  *
- * See also: [Request]
+ * @see: [Request]
  */
 class Response {
     private var _status: Int? = null
     private var _body: ByteArray? = null
 
-    /** See also: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status */
+    /** @see: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status */
```
[the remark this answers is not in the document]
Comment (tungbq) [reply backend/kobweb-api/src/main/kotlin/com/varabyte/kobweb/api/http/Response.kt]: Updated

File: frontend/compose-html-ext/src/jsMain/kotlin/com/varabyte/kobweb/compose/dom/ElementRefScope.kt [hunk 3]
```diff
@@ -125,7 +125,7 @@ fun <TElementSrc : Element, TElementDst : Element> ElementScope<TElementSrc>.reg
  * }
  * ```
  *
- * See also: [disposableRef], [refScope]
+ * @see: [disposableRef], [refScope]
```
[the remark this answers is not in the document]
Comment (tungbq) [reply frontend/compose-html-ext/src/jsMain/kotlin/com/varabyte/kobweb/compose/dom/ElementRefScope.kt]: Updated

File: frontend/kobweb-core/src/jsMain/kotlin/com/varabyte/kobweb/navigation/Router.kt [hunk 4]
```diff
@@ -262,7 +262,7 @@ class Router {
      *
      * That way, either the router handles the navigation or the browser does.
      *
-     * See also: [navigateTo], which, if called, handles the external navigation for you.
+     * @see: [navigateTo], which, if called, handles the external navigation for you.
```
[the remark this answers is not in the document]
Comment (tungbq) [reply frontend/kobweb-core/src/jsMain/kotlin/com/varabyte/kobweb/navigation/Router.kt]: Updated

**Comment by tungbq** [comment]
@bitspittle I've update the code follow your comments. Also base on top of 0.13.12. Please review again, thanks.

**Review by bitspittle — APPROVED** [review]
Thank you!

**Comment by bitspittle** [comment]
FYI you might be curious to see I did some quick followup tweaks to your changes. Thanks to your work, this was trivial!

https://github.com/varabyte/kobweb/commit/132c3fe5271907e8d484774b87843cace8b76574

I used regex find-replace to convert `@see [asdf]` to `@see asdf`. It's such a cool feature:

regex-find-replace

(***NOTE:** To use it, you need to make sure the "regex mode" button is checked in the find/replace window, which got cut off in my screenshot)*

Appreciate your change. Let me know if you ever decide to explore more of the codebase and want some help navigating.

**Pull request merged**